lib/charms/hpc_libs/v0/slurm_ops.py

```diff
@@ -674,6 +634,28 @@ def _create_state_save_location(self) -> None:
     def _apply_overrides(self) -> None:
         """Override defaults supplied provided by Slurm Debian packages."""
         match self._service_name:
+            case "sackd":
+                _logger.debug("overriding default sackd service configuration")
+                config_override = Path(
+                    "/etc/systemd/system/sackd.service.d/10-sackd-config-server.conf"
+                )
+                config_override.mkdir(parents=True, exist_ok=True)
```

Comment:
Spotted in testing: this should be `config_override.parent.mkdir`.

Reply:
Fixed in latest batch of commits.